Game/Scripts/Player.as

```diff
@@ -0,0 +1,24 @@
+class Player{
+    Hub @hub;
+    Entity @self;
+    Entity @cart;
+    vec2 Cursor;
+    Player(Entity @entity){
+        @hub = Managers();
+        @self = @entity;
+        @cart = @self.GetParent();
+        RegisterUpdate();
+        Cursor = vec2(0,0);
+    }
+    
+    //Update carts movements and send it's position to Player Script.
+    void Update(float i){
+        self.position = vec3(cart.position.x, cart.position.y+4, cart.position.z+1);
```

Comment:
This will make the player's and cart's positions desync as time goes on. Player is already relative to cart.